app/javascript/components/Pages/modals/CelebrateModal.js

```diff
@@ -0,0 +1,52 @@
+import React, {Fragment} from "react";
+import Form from "react-bootstrap/Form";
+import Modal from "react-bootstrap/Modal";
+import {apiRequest} from "../../requests/axios_requests";
+import xClose from "../../../../assets/images/sys_svg/x-close.svg";
+
+const CelebrateModal = ({ show, setShow, steps, celebrateShoutout, setCelebrateShoutout, saveDataToDb, goToRecognitionPage }) => {
+  const handleClick = (e) => {
+    setCelebrateShoutout(Object.assign({}, celebrateShoutout, {not_ask: e.target.checked}));
+  };
+  const handleMakeVisible = () => {
+    const dataSend = { shoutout: {visible: true, not_ask: celebrateShoutout.not_ask} }
+    const dataFromServer = (shoutout) => {
+      saveDataToDb(steps, {shoutout_id: shoutout.id})
+    }
+    const url = '/api/v1/shoutouts/'
+    const id = celebrateShoutout?.id
+    apiRequest("PATCH", dataSend, dataFromServer, ()=>{}, `${url}${id}`).then();
+    goToRecognitionPage()
+  };
+
+  return <Fragment>
+    {show && <div className='backdrop celebrate-modal' /> }
+    <Modal show={show} onHide={() => {setShow(false)}} animation={true} className='modal-dialog-celebrate'>
+      <img src={xClose} className='position-absolute x-close' onClick={() => {setShow(false)}}/>
+      <Modal.Body>
+        <Form>
+          <Form.Group className="mb-3">
+            <div className='fs-5 fw-bold'>
+              Including <span className='red-violet'>@</span>Shoutouts will<br/>allow other team member(s)<br/>to see your response.<br/><br/>
+              <div className='fs-6 muted'>Are you ok with it?</div>
```

Comment:
Please change "Are you ok with it?" to "Is this OK?"

Reply:
Thanks, I checked   - Code Climate issues are not relevant.